modules/claims_api/app/controllers/claims_api/v1/forms/disability_compensation_controller.rb

```diff
@@ -153,6 +154,13 @@ def validate_form_526_submission_claim_date!
           raise ::Common::Exceptions::InvalidFieldValue.new('claimDate', form_attributes['claimDate'])
         end
 
+        def validate_form_526_claimant_certification!
+          unless form_attributes['claimantCertification'] == true
+            raise ::Common::Exceptions::InvalidFieldValue.new('claimantCertification',
+                                                              form_attributes['claimantCertification'])
+          end
```

Comment:
nit: something ive gotten accustomed to with ruby is favoring guard clauses in these types of situations
i started doing it because rubocop's default favors them
https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/GuardClause

so this could turn into something like
```
return unless form_attributes['claimantCertification'] == true

raise ::Common::Exceptions::InvalidFieldValue.new('claimantCertification',
                                                                                  form_attributes['claimantCertification'])
```
its a minor thing but i personally think it helps readability in a lot of situations by reducing nesting

Reply:
that is a ruby convention that I still have to remind myself of 🙃 I've got no problem updating that